### PR TITLE
Motion synthesis and test set prediction error

### PR DESCRIPTION
**Pull request by moduIo**
Adds
* Sequence generator
* Test set error

### CODE REVIEW

File: utils/args.py [hunk 1]
```diff
@@ -46,4 +58,51 @@ def parse_main_args():
...
+
+    parameters = parser.add_argument_group("Model Parameters")
+    parameters.add_argument("-b", "--batch_size", default=32, type=int)
+    parameters.add_argument("-emb", "--embedding_dim", default=128, type=int)
+    parameters.add_argument("-ep", "--epochs", default=100, type=int)
+    parameters.add_argument("-nh", "--nhead", default=8, type=int)
+    parameters.add_argument("-enc", "--nlayers", default=8, type=int)
+    parameters.add_argument("-ff", "--feedforward_dim", default=256, type=int)
+    parameters.add_argument("-do", "--dropout", default=0.1, type=float)
+
```
Comment (jrevansjr) [inline utils/args.py:106]: My commit adds command line args here

File: utils/args.py [hunk 2]
```diff
@@ -46,4 +58,51 @@ def parse_main_args():
     parameters.add_argument("-ff", "--feedforward_dim", default=256, type=int)
     parameters.add_argument("-do", "--dropout", default=0.1, type=float)
```
Comment (jrevansjr) [inline utils/args.py:60]: My commit adds command line args here

File: utils/loss.py [hunk 3]
```diff
@@ -46,4 +47,5 @@ def compute_validation_loss(args, model, datasets, criterion, device, mask):
             epoch_val_loss += loss.item()
             break  # TODO: Delete this
 
+    model.train()
```
Comment (jrevansjr) [inline utils/loss.py:50]: My commit changes quite a bit here but good catch with the model. eval
